features/accessibility/provider.feature

```diff
@@ -188,10 +188,10 @@ Feature: Provider accessibility
 #    And the page is accessible
     Then I choose 'No'
     Then I click 'Save and continue'
-    Then I should be on the 'policy_disregards' page showing 'schemes or charities'
-    Then I select "None of these"
-    And the page is accessible
-    Then I click 'Save and continue'
+#    Then I should be on the 'policy_disregards' page showing 'schemes or charities'
+#    Then I select "None of these"
+#    And the page is accessible
+#    Then I click 'Save and continue'
```

Comment:
left overr comments

Reply:
Annoyingly they need to be uncommented tomorrow!
>These were written for as expected behaviour and will be correct
after the 9th January
Branching logic in rspec handles the logic so these should
be restored on the 8th January

Reply:
Working on a fix for these... It's something that will probably still need some love after the update!